docs/manual/troubleshooting.tex

```diff
@@ -1073,6 +1073,30 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
 \end{enumerate}
 
 
+\subsection{Configure Eclipse to edit the Checker Framework\label{building-eclipse}}
+
+\begin{enumerate}
+\item Follow the instructions earlier in Section~\ref{build-source} to
+clone and build all projects from their sources.
+\item Download Eclipse from
+the \href{https://www.eclipse.org/downloads}{official Eclipse website}
+and install it.
+\item Run Eclipse and set the workspace to \code{\$JSR308}.
```

Comment:
Is this the literal 7 characters `$JSR308`, or the value of that environment variable?  (Similar question below.)